content/projects/git-exercises/_index.md

```diff
@@ -407,9 +407,9 @@ If you are curious now, spend some time googling git branching strategies. We us
 
 ## Instructions For Reviewers
 
-- Ensure that there are 3 branches: main, history, and milkshake-flavours
-- The main branch should consist of 4 files: .gitignore, README.md, history.txt, and milkshakes.md
-- The history branch should consist of 3 files. Consider that main should be merged into history before the .gitignore file is created.
-- The milkshake-flavours branch should consist of 2 files.
-- The repo should have a minimum of 9 commits. Take a look at the network diagram or graph to check that the branches are present and have been merged into main. Also, check that main has been merged back into history.
-- Ensure that there are no merge conflicts on the README.md file.
+- Check if multiple branches exist, to ensure recruits understand how to create branches.
+- Check if the recruit understands how merging works, by ensuring that the main branch has files merged into it. You can also take a look at the network diagram to check for merging.
+- Check if there are multiple commits.
```

Comment:
Is this point necessary? Is it not common sense for there to be multiple commits?